azure-iot-hub/azure/iot/hub/digital_twin_service_client.py

```diff
@@ -0,0 +1,44 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from .auth import ConnectionStringAuthentication
+
+from .protocol.iot_hub_gateway_service_ap_is20190701_preview import (
+    IotHubGatewayServiceAPIs20190701Preview as pl_client,
+)
+from .protocol.models.digital_twin_interfaces import DigitalTwinInterfaces
+from .protocol.models.digital_twin_interfaces_patch import DigitalTwinInterfacesPatch
+
+DigitalTwin = DigitalTwinInterfaces
+DigitalTwinPatch = DigitalTwinInterfacesPatch
```

Comment:
You can actually do this in the import statement itself

e.g.

`from .protocol.models import DigitalTwinInterfaces as DigitalTwin`